src/input.scss

```diff
@@ -32,11 +32,12 @@ $block: #{$fd-namespace}-input;
   }
 
   &::placeholder {
-    @include fd-reset();
-
     line-height: $fd-form-input-height;
     font-style: italic;
-    color: var(--sapField_PlaceholderTextColor);
```

Comment:
Do we wan't to remove `color` from the placeholder?

Reply:
No we don't

Reply:
Actually we do! The color of the placeholder is set in _forms.scss

Reply:
font-style is also not necessary

Reply:
right, removed by mistake, revert is comming